File: medium/pro-carrot-cake/CarrotCake.java
import java.util.Scanner;

public final class CarrotCake {
    public static void main(String[] args) {
        final var scanner = new Scanner(System.in);
        final var carrotsForCake = 7;
        final var carrots = scanner.nextInt();
        final var boxes = scanner.nextInt();
        
        final var leftOverCarrots = carrots % boxes;
        final var isEnoughForCake = leftOverCarrots >= carrotsForCake;
        
        if (isEnoughForCake) {
            System.out.print("Cake Time");
        } else {
            final var needMoreCarrots = carrotsForCake - leftOverCarrots;
            System.out.printf("I need to buy %d more", needMoreCarrots);
        }
    }
}

File: medium/pro-carrot-cake/carrot_cake.c
#include <stdio.h>

int main() {
    int const carrotsForCake = 7;
    int carrots, boxes;
    scanf("%d\n%d", &carrots, &boxes);
    
    int const leftOverCarrots = carrots % boxes;
    int const isEnoughForCake = leftOverCarrots >= carrotsForCake;
    
    if (isEnoughForCake) {
        printf("Cake Time");
    } else {
        int const needMoreCarrots = carrotsForCake - leftOverCarrots;
        printf("I need to buy %d more", needMoreCarrots);
    }
    return 0;
}
